commit 1babf63444b7b7f3c84e2561fbdecd5fd5a0e36f
Author: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date:   Tue Mar 24 09:06:29 2020 +0100

    src/evemu.c: fix build with kernels < 4.16
    
    Commit 30973c0b fixed the build with kernels >= 4.16 but as a side
    effect, build with kernels < 4.16 is now failing on:
    
    evemu.c:892:18: error: ‘struct input_event’ has no member named ‘input_event_usec’
       tv.tv_usec = ev->input_event_usec;
                      ^~
    
    Fix this error by defining input_event_sec and input_event_usec if
    needed
    
    Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

diff --git a/src/evemu.c b/src/evemu.c
index 42dbe1d..2f457c1 100644
--- a/src/evemu.c
+++ b/src/evemu.c
@@ -63,6 +63,14 @@
 
 #define SYSCALL(call) while (((call) == -1) && (errno == EINTR))
 
+#ifndef input_event_sec
+#define input_event_sec time.tv_sec
+#endif
+
+#ifndef input_event_usec
+#define input_event_usec time.tv_usec
+#endif
+
 enum error_level {
 	INFO,
 	WARNING,
